adi/adis16475.py

```diff
@@ -65,6 +65,19 @@ def __init__(self, uri="", device_name="adis16505-2"):
             self._ctrl = self._ctx.find_device(device_name)
             self._rxadc = self._ctx.find_device(device_name)
 
+        if self._ctrl is None:
+            print(
+                "No device found with device_name = "
+                + device_name
+                + ". Searching for a device found in the compatible list."
+            )
+            for i in compatible_parts:
+                self._ctrl = self._ctx.find_device(i)
+                self._rxadc = self._ctx.find_device(i)
+                if self._ctrl is not None:
+                    print("Fond device = " + i + ". Will use this device instead.")
+                    break
+
```

Comment:
Need a check to make sure a device is found

Reply:
I added an exception in case no device is found